cpp/CMakeLists.txt

```diff
@@ -177,7 +179,8 @@ add_library(raft::raft ALIAS raft)
 
 target_include_directories(raft INTERFACE
         "$<BUILD_INTERFACE:${RAFT_SOURCE_DIR}/include>"
-        "$<INSTALL_INTERFACE:include>")
+        "$<INSTALL_INTERFACE:include>"
+        "$<BUILD_INTERFACE:${CUTLASS_INCLUDE_DIR}>")
```

Comment:
Drop this line and instead update the `target_link_libraries` to also link to `nvidia::cutlass::cutlass`.